docs/content/users/quickstart.md

```diff
@@ -196,6 +196,18 @@ The legacy type `drupal` will be interpreted as the latest stable version of Dru
     ddev launch $(ddev drush uli)
     ```
 
+=== "Drupal CMS"
+
+    ```bash
+    mkdir my-drupal-cms && cd my-drupal-cms
+    ddev config --project-type=drupal11 --docroot=web
+    ddev composer create --stability="RC" drupal/cms
+    ddev drush site:install --account-name=admin --account-pass=admin -y
+    ddev launch
+    # or automatically log in with
+    ddev launch $(ddev drush uli)
```

Comment:
We should remove the `drush site:install` stuff. Generally we want people to be sent to the UI installer; using Drush skips over some steps (like selecting content type recipes).